### PR TITLE
Restore font style of sponsored cAPI component headings

### PR DESCRIPTION
**Pull request by jbreckmckye**
This fixes a side effect of [#10080 - Add media icon to merchandising component items](https://github.com/guardian/frontend/pull/10080), where the font weight / size for sponsored cAPI component headings was reduced:

**Before #10080 (good):**
before

**After #10080 (bad):**
after

In the original PR, I changed a h2.fc-item__header into a h2.fc-item__title, because media icons need to live inside a title. I've now learned that fc-item__title needs to live inside an fc-item__header, so wrapped the h2 and restored a style that keeps the font sizes in order for merchandizing component items.

The font style of sponsored component headings is restored:

image

And the media icons added in #10080 still appear as expected:

image

@Calanthe @uplne

### CODE REVIEW

File: commercial/app/views/contentapi/items.scala.html [hunk 1]
```diff
@@ -44,10 +44,12 @@ <h3 class="ad-slot--paid-for-badge__header">@optSponsorLabel:</h3>
                                             @fragments.items.elements.facia_cards.image(trailPictureContainer, true)
                                         }
                                         <div class="fc-item__content">
-                                            <h2 class="fc-item__title">
-                                                @insertItemMediaIcon(item)
-                                                @item.webTitle
-                                            </h2>
+                                            <div class="fc-item__header">
```
Comment (uplne) [inline commercial/app/views/contentapi/items.scala.html]: Is it necessary to add another div? Can't you include that font inside fc-item__title with the same result?

Comment (jbreckmckye) [inline commercial/app/views/contentapi/items.scala.html]: __header comes with quite a few styles; I guess we could inject them into __title for this component, but this was the structure used elsewhere - like the fronts - and I suppose I just thought it was best to use it the same way here.

**Comment by uplne** [comment]
:+1:

**Comment by jbreckmckye** [comment]
With TeamCity's builds paused I've run my own tests. Everything looks fine, so I'll be merging this into master now.

**Pull request merged**